apps/web-giddh/src/app/shared/header/components/export-master-dialog/export-master-dialog.component.ts

```diff
@@ -0,0 +1,122 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { Router } from '@angular/router';
+import { ExportBodyRequest } from 'apps/web-giddh/src/app/models/api-models/DaybookRequest';
+import { LedgerService } from 'apps/web-giddh/src/app/services/ledger.service';
+import { ToasterService } from 'apps/web-giddh/src/app/services/toaster.service';
+import { ReplaySubject, takeUntil } from 'rxjs';
+import { ManageGroupsAccountsComponent } from '../new-manage-groups-accounts/manage-groups-accounts.component';
+import { Store } from '@ngrx/store';
+import { AppState } from 'apps/web-giddh/src/app/store';
+import { GroupWithAccountsAction } from 'apps/web-giddh/src/app/actions/groupwithaccounts.actions';
+
+@Component({
+  selector: 'app-export-master-dialog',
+  templateUrl: './export-master-dialog.component.html',
+  styleUrls: ['./export-master-dialog.component.scss']
+})
+export class ExportMasterDialogComponent implements OnInit {
+  /** Form Group for export  form */
+  public exportForm: any;
+  /** True if api call in progress */
+  public isLoading: boolean = false;
+  /** To destroy observers */
+  public destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
+  /* This will hold local JSON data */
+  public localeData: any = {};
+  /* This will hold common JSON data */
+  public commonLocaleData: any = {};
+
+  constructor(
+    @Inject(MAT_DIALOG_DATA) public inputData,
+    private ledgerService: LedgerService,
+    private toaster: ToasterService,
+    private router: Router,
+    private dialogRef: MatDialogRef<ManageGroupsAccountsComponent>,
+    private store: Store<AppState>,
+    private groupWithAccountsAction: GroupWithAccountsAction
+  ) { }
+
+  ngOnInit() {
+  }
+
+
+
+  public exportMaster(): void {
+    let exportRequest: ExportBodyRequest = new ExportBodyRequest();
+    exportRequest.exportType = this.inputData?.exportType;
+    exportRequest.columnsToExport = [];
+    const formValue = this.exportForm;
```

Comment:
```suggestion
    const formValue = this.exportFormValue;
```

Correct in overall where it is using also check in parent for formValue

Reply:
Done